pkg/chart/chart.go

```diff
@@ -71,6 +72,7 @@ type Helm interface {
 	LintWithValues(chart string, valuesFile string) error
 	Install(chart string, namespace string, release string) error
```

Comment:
This method is now unused; we may want to remove this or refactor to replace `InstallWithValues`.

Reply:
I'd say remove and refactor

Reply:
We should use the same logic for `Lint`.

Reply:
I've removed `Lint` and `Install`. That seems like the smallest possible change, but we could also replace these function implementations (e.g. rename `InstallWithValues` --> `Install`) instead.

As an aside, we might consider moving `Helm`, `Kubectl`, etc interfaces and implementations to internal packages in order to avoid breaking changes when doing these sorts of refactors in the future.